merge_mafs/merge_mafs.py

```diff
@@ -0,0 +1,55 @@
+import sys
+import argparse
+
+parser = argparse.ArgumentParser(description = 'This program reads in 2 MAF files and merges them to one.')
+parser.add_argument("maf_file1", type=str, help="Input MAF1")
+parser.add_argument("maf_file2", type=str, help="Input MAF2")
+parser.add_argument("merged_maf", type=str, help="Output merged MAF filename")
+args = parser.parse_args()
+
+file_header = list();
+
+# Check if columns in the two input maf files are in same order	
+
+try:
+	with open(args.maf_file1,'r') as file:
+		header = file.readline()
+		file_header.append(header)
+except IOError:
+	print args.maf_file1,"file does not exist."
+	sys.exit()
+	
+try:
+	with open(args.maf_file2,'r') as file:
+		header = file.readline()
+		file_header.append(header)
+except IOError:
+	print args.maf_file2,"file does not exist."
+	sys.exit()
+
+header1 = file_header[0]
+header2 = file_header[1]
+
+if header1 != header2:
+	print("You are trying to concatenate MAF files with columns of differnt order. Exiting..")
```

Comment:
hmm I'm thinking that it might be easiest to hack the merge script to merge your mafs into a single file. 

If you're merging 2 MAFs with the exact same header then you could do something like this:

1. copy MAF1 header and contents, ignoring comments, and give it the new merged MAF filename
```grep -v "^#" maf1 > mergedmaf```
2. concatenate MAF2 contents (ignoring comments and header) to the new merged MAF file
```grep -v -E "^#|^Hugo" maf2 >> mergedmaf```
and that would give you a merged maf. 

To check if the headers are identical, you could:

```
grep -v "^#" maf1 | head -1 > header1
grep -v "^#" maf2 | head -1 > header2
diff header1 header2
```
If identical then the `diff` won't return anything. 

You could basically reduce this python script into a simple shell script that does all of these steps I outlined or just store them in a README for your reference.